kategory/src/test/kotlin/kategory/data/ListKWTest.kt

```diff
@@ -1,24 +1,24 @@
 package kategory.data
 
 import io.kotlintest.KTestJUnitRunner
-import io.kotlintest.matchers.shouldBe
+import kategory.Eq
 import kategory.ListKW
-import kategory.NonEmptyList
+import kategory.TraverseLaws
 import kategory.UnitSpec
-import kategory.binding
 import org.junit.runner.RunWith
 
 @RunWith(KTestJUnitRunner::class)
 class ListKWTest : UnitSpec() {
     init {
-
-        testLaws(kategory.MonadLaws.laws(kategory.NonEmptyList, kategory.Eq.any()))
+        val applicative = ListKW.applicative()
+        testLaws(kategory.MonadLaws.laws(ListKW, kategory.Eq.any()))
         testLaws(kategory.SemigroupKLaws.laws(
                 kategory.ListKW.semigroupK(),
-                kategory.ListKW.applicative(),
+                applicative,
                 kategory.Eq.any()))
+        testLaws(TraverseLaws.laws(ListKW.traverse(), applicative, { applicative.pure(it) }, Eq.any()))
```

Comment:
The second parameter, `applicative` , has to be of another type for simplicity. Try Option instead.